src/Microsoft.AspNetCore.Blazor.Templates/content/BlazorServerSide-CSharp/.template.config.src/template.json

```diff
@@ -0,0 +1,114 @@
+      "path": "BlazorServerSide-CSharp.sln"
+    },
+    {
+      "condition": "(OS == \"Windows_NT\" && HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+      "path": "BlazorServerSide-CSharp.Server\\BlazorServerSide-CSharp.Server.csproj"
+    },
+    {
+      "condition": "(OS != \"Windows_NT\" && HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+      "path": "BlazorServerSide-CSharp.Server/BlazorServerSide-CSharp.Server.csproj"
+    },
+    {
+      "condition": "(OS == \"Windows_NT\" && HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+      "path": "BlazorServerSide-CSharp.Client\\BlazorServerSide-CSharp.Client.csproj"
+    },
+    {
+      "condition": "(OS != \"Windows_NT\" && HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+      "path": "BlazorServerSide-CSharp.Client/BlazorServerSide-CSharp.Client.csproj"
+    }
+  ],
+  "shortName": "blazorserverside",
+  "sourceName": "BlazorServerSide-CSharp",
+  "sources": [
+    {
+      "source": "./",
+      "target": "./",
+      "exclude": [
+        ".template.config/**"
+      ],
+      "modifiers": [
+        {
+          "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+          "exclude": [
+            "*.sln"
+          ]
+        }
+      ]
+    }
+  ],
+  "symbols": {
+    "Framework": {
+      "type": "parameter",
+      "description": "The target framework for the project.",
+      "datatype": "choice",
+      "choices": [
+        {
+          "choice": "netcoreapp2.1",
+          "description": "Target netcoreapp2.1"
+        }
+      ],
+      "replaces": "netcoreapp2.1",
+      "defaultValue": "netcoreapp2.1"
+    },
+    "HostIdentifier": {
+      "type": "bind",
+      "binding": "HostIdentifier"
+    },
+    "IncludeCustomRestoreSourcesSymbol": {
+      "type": "parameter",
+      "datatype": "bool",
+      "description": "If set, includes restore sources for the Blazor MyGet feed.",
+      "defaultValue": "true"
+    },
+    "TemplateBlazorVersionSymbol": {
+      "type": "parameter",
+      "datatype": "string",
+      "description": "Specifies which version of Blazor packages to use.",
+      "replaces": "$(TemplateBlazorPackageVersion)",
+      "defaultValue": "0.5.0-preview1-t000"
+    },
+    "skipRestore": {
+      "type": "parameter",
+      "datatype": "bool",
+      "description": "If specified, skips the automatic restore of the project on create.",
+      "defaultValue": "false"
+    }
+  },
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "postActions": [
+    {
+      "condition": "(!skipRestore)",
+      "description": "Restore NuGet packages required by this project.",
+      "manualInstructions": [
+        {
+          "text": "Run 'dotnet restore'"
+        }
+      ],
+      "actionId": "210D431B-A78B-4D2F-B762-4ED3E3EA9025",
+      "continueOnError": true
+    }
+  ]
+}
```

Comment:
It's difficult to read all this and be sure it's correct. I'm going to assume you copy-pased one of the existing ones and made the right edits.